libbindgen/src/ir/function.rs

```diff
@@ -147,6 +147,7 @@ impl FunctionSig {
         };
         let mut args: Vec<_> = match cursor.kind() {
             CXCursor_FunctionDecl |
+            CXCursor_Constructor |
             CXCursor_CXXMethod => {
```

Comment:
Yeah, we're not doing it. We can without too much trouble though.

When we do, there's no point in doing that, though (we could, I guess?). But destructors have no arguments.